assets/prometheus-adapter/deployment.yaml

```diff
@@ -82,6 +82,14 @@ spec:
             drop:
             - ALL
           readOnlyRootFilesystem: true
+        startupProbe:
+          failureThreshold: 18
+          httpGet:
+            path: /livez
```

Comment:
Is it better to use `/readyz`? 
If a pod is fully started, we may consider it ready to serve traffic. In most cases `/livez` and `/readyz` return 200 OK within the same second though.

Reply:
I used `/livez` because according to the k8s docs once a pod passes the startupProbe then the livenessProbe takes over, hence I used the same endpoint as the livenessProbe.